spec/spec/event.js

```diff
@@ -71,39 +72,32 @@ describe('Event', function() {
       SVG.off(el, 'event', action)
     })
     it('attaches multiple handlers on different element', function() {
-      var listenerCnt = SVG.listeners.length
-
-      var rect2 = draw.rect(100,100);
-      var rect3 = draw.rect(100,100);
+      var rect2 = draw.rect(100, 100)
+      var rect3 = draw.rect(100, 100)
 
       rect.on('event', action)
       rect2.on('event', action)
       rect3.on('event', function(){ butter = 'melting' })
       rect3.on('event', action)
 
-      expect(Object.keys(SVG.listeners[SVG.handlerMap.indexOf(rect.node)]['event']['*']).length).toBe(1)  // 1 listener on rect
-      expect(Object.keys(SVG.listeners[SVG.handlerMap.indexOf(rect2.node)]['event']['*']).length).toBe(1) // 1 listener on rect2
-      expect(Object.keys(SVG.listeners[SVG.handlerMap.indexOf(rect3.node)]['event']['*']).length).toBe(2) // 2 listener on rect3
-
-      expect(SVG.listeners.length).toBe(listenerCnt + 3)                                                  // added listeners on 3 different elements
+      expect(Object.keys(rect.events['event']['*']).length).toBe(1)  // 1 listener on rect
```

Comment:
Whats this star doing?

Reply:
The star is a wildcard namespace. So every event not bound to a particular namespace will get saved under the wildcard namespace

Reply:
Oh okay, sounds fair to me.

Reply:
Maybe though, it should be clearer? Maybe something like 'default'?

Reply:
I just wanted to take something which wouldnt be used by the user on accident